templates/survey/stats.html

```diff
@@ -275,6 +275,13 @@ <h4>
                             {% endfor %}
                         </select>
                     </div>
+                    <div class="col-md-auto" id="select-section-graphType" style="display:none">
+                        <label for="section_graphType">{{ _('Graph Type')}}</label>
+                        <select id="section_graphType">
+                            <option value="radar" selected>{{ _('Radar')}}</option>
+                            <option value="bubble">{{ _('Bubble')}}</option>
```

Comment:
Is this necessary? The words are more "technical", I mean, if someone is looking at statistics, they may already be familiar with these terms. Otherwise, I can add the translation via Weblate for sure.